### PR TITLE
Feature: Lock API MVP

### PR DESCRIPTION
**Pull request by johnkariuki**

### CODE REVIEW

**Comment by kabanishe23** [comment]
@johnkariuki Can you please handle my previous comment?

**Comment by johnkariuki** [comment]
@kabanishe23 I will add this up. I have been a bit held up at the moment.

**Comment by johnkariuki** [comment]
@kabanishe23 Pushed

File: services/locks.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      name: req.body.name,
+      user: req.user.id
+    });
+  },
+  get(req) {
+    return Lock.findOne({
+      where: {
+        $or: [
+          { id: { $eq: req.params.lockId } },
+          { macId: { $eq: req.params.lockId } }
```
Comment (kabanishe23) [inline services/locks.js]: @johnkariuki shouldn't it be **macId** in params?

Comment (johnkariuki) [reply services/locks.js]: @kabanishe23 
this is actually a workaround against the already existing functionality.
I first created an `id` search param but then when I added the `macId` to the same endpoint, I maintained the param name.

Would it make sense if it was named `filter` or `searchParam` instead?

Comment (kabanishe23) [reply services/locks.js]: I guess filter will work better

Comment (johnkariuki) [reply services/locks.js]: Makes sense. I have updated it accordingly.

**Pull request merged**